packages/php-wasm/compile/php/Dockerfile

```diff
@@ -381,13 +384,15 @@ RUN cd /root && \
 	git apply --no-index /root/php${PHP_VERSION:0:3}*.patch -v && \
 	chmod +x /root/apply-mysqlnd-patch.sh && \
 	/root/apply-mysqlnd-patch.sh && \
-	( [[ "${PHP_VERSION:0:3}" == '8.3' ]] && \
-		git apply --no-index /root/php-chunk-alloc-zend-assert-8.3.patch -v || \
-		( [[ "${PHP_VERSION:0:3}" == '8.4' ]] && \
-			git apply --no-index /root/php-chunk-alloc-zend-assert-8.4.patch -v || \
-			git apply --no-index /root/php-chunk-alloc-zend-assert.patch -v \
-		) \
-	) && \
+	if [[ "${PHP_VERSION:0:3}" < '8.3' ]]; then \
+		git apply --no-index /root/php-chunk-alloc-zend-assert.patch -v; \
+	elif [[ "${PHP_VERSION:0:3}" == '8.3' ]]; then \
+		git apply --no-index /root/php-chunk-alloc-zend-assert-8.3.patch -v; \
+	elif [[ "${PHP_VERSION:0:3}" == '8.4' ]]; then \
+		git apply --no-index /root/php-chunk-alloc-zend-assert-8.4.patch -v; \
+	elif [[ "${PHP_VERSION:0:3}" == '8.5' ]]; then \
+		git apply --no-index /root/php-chunk-alloc-zend-assert-8.5.patch -v; \
+	fi && \
```

Comment:
While adding the PHP 8.5 patch, I noticed that patches for PHP versions before 8.3weren't applied.